worlds/sc2/Items.py

```diff
@@ -68,175 +213,807 @@ def get_full_item_list():
-    ItemNames.HERCULES: ItemData(615 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 15, SC2Race.TERRAN, classification=ItemClassification.progression),
-    ItemNames.CELLULAR_REACTOR: ItemData(616 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 16, SC2Race.TERRAN),
-    ItemNames.PROGRESSIVE_REGENERATIVE_BIO_STEEL: ItemData(617 + SC2WOL_ITEM_ID_OFFSET, "Progressive Upgrade", 4, SC2Race.TERRAN, quantity=2),
-    ItemNames.HIVE_MIND_EMULATOR: ItemData(618 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 18, SC2Race.TERRAN, ItemClassification.progression),
-    ItemNames.PSI_DISRUPTER: ItemData(619 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 19, SC2Race.TERRAN, classification=ItemClassification.progression),
+    ItemNames.VALKYRIE_ENHANCED_CLUSTER_LAUNCHERS: ItemData(379 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 17,
+                                                            SC2Race.TERRAN, parent_item="Valkyrie", origin={"ext"},
+                                                            description="""
+                                                            Valkyrie fires 2 additional rockets each volley.
+                                                            """),
+    ItemNames.VALKYRIE_SHAPED_HULL: ItemData(380 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 20, SC2Race.TERRAN,
+                                             classification=ItemClassification.filler, parent_item="Valkyrie",
+                                             origin={"ext"},
+                                             description="""
+                                             Increases Valkyrie life by 50.
+                                             """),
+    ItemNames.VALKYRIE_BURST_LASERS: ItemData(381 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 21, SC2Race.TERRAN,
+                                              parent_item="Valkyrie", origin={"ext"},
+                                              description="""
+                                              Valkyries are equipped with Burst Lasers to attack ground units.
+                                              """),
+    ItemNames.VALKYRIE_AFTERBURNERS: ItemData(382 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 22, SC2Race.TERRAN,
+                                              classification=ItemClassification.filler, parent_item="Valkyrie",
+                                              origin={"ext"},
+                                              description="""
+                                              Allows Valkyries a quick burst of movement speed.
+                                              """),
+
+    ItemNames.BUNKER: ItemData(400 + SC2WOL_ITEM_ID_OFFSET, "Building", 0, SC2Race.TERRAN,
+                               classification=ItemClassification.progression,
+                               description="""
+                               Defensive structure. Allows to garrison infantry to attack from inside.
```

Comment:
"Allows to load" sounds like shorthand in English rather than a proper sentence.
Official description:
> Defensive structure. Able to protect up to <d ref="Abil,BunkerTransport,TotalCargoSpace"/> infantry unit slots. Units inside the Bunker have +1 range to their attacks. Bunkers can be salvaged.

My recommendation:
> Defensive structure. Able to load infantry units, giving them +1 range to their attacks.